CHANGELOG.md

```diff
@@ -4,7 +4,53 @@
 
 #### Changed
 
-- Internal CPU mining now injects nonce bytes at the end.
+## 1.7 (2019-03-26)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-04-02T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2020-04-30T00:00:00Z**.
+
+Breaking validation changes:
+
+* Compute epoch time based on parent header. This change won't affect any
+  any users of Chainweb. The change will become effective at block height
+  452820. (#977)
+
+* Validate transaction creation time and expiration with respect to creation
+  of the parent block header. This change brings the checks for transaction
+  validity in line with the time used during Pact validation. (#935, #942)
+
+  There is a 90 second leniency applied for transactions to become valid. (#973)
+
+  This change will become effective at chain block height 449940. In the time
+  before the change becomes active users may experience that new nodes reject
+  their transactions. This can be mitigated by using a negative offset for the
+  creation of about 2 minutes.
```

Comment:
Negative offset == in the past, or the future?

Reply:
`- 2min`. @mightybyte what is the format in Chainweaver to specify tx creation time?

Reply:
- Put a link in for the block height code line ("see here for block height") instead of duping it here in case we have a last minute change. 
- this should be one bullet para with #973 at the end

Reply:
Chainweaver sets the creation time automatically.

Reply:
Can it be overwritten by the user?

Reply:
@slpopejoy using a link is tricky. Either it would have to point to a particular commit, in which case we would have to change the link if we change the code. Or it would point to master, in which case it may break if we change the code in the future (e.g. consolidate old guards, as we did before).

I added a reminder for double checking these values to the TODO list in the release-candidate PR.